backend/Squirrel.ConsoleApp/Program.cs

```diff
@@ -1,9 +1,27 @@
-﻿namespace Squirrel.ConsoleApp;
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+
+namespace Squirrel.ConsoleApp;
 
 internal class Program
 {
     static void Main(string[] args)
     {
-        Console.WriteLine("Hello, World!");
+        var config = new ConfigurationBuilder()
+            .AddJsonFile("appsettings.json")
+            .Build();
+
+        var baseUrl = config.GetSection("WebServerSettings")["BaseUrl"];
+        
+        CreateHostBuilder(args, baseUrl).Build().Run();
     }
+
+    public static IHostBuilder CreateHostBuilder(string[] args, string baseUrl) =>
+        Host.CreateDefaultBuilder(args)
```

Comment:
@Anatolii-Ivanov @alinaSielina how we are going to run/host ConsoleApp?

Reply:
It will listen to user computers port and respond to actions to it (Also, it will happen for short period of time)